### PR TITLE
Added auraloss loss modules for Multi-Resolution and Sum/Difference STFT losses

### PR DESCRIPTION
**Pull request by zqevans**

### CODE REVIEW

File: oobleck/losses.py [hunk 1]
```diff
@@ -38,3 +38,22 @@ def forward(self, inputs: TensorDict) -> torch.Tensor:
...
         kl = (mean.pow(2) + var - logvar - 1).sum(1).mean() * self.beta_kl
         return {"generator_loss": l1 + kl, "kl_divergence": kl}
+
+
+class AuralossWrapper(nn.Module):
+
+    def __init__(self, 
+                 input_key: str, 
+                 output_key: str,
+                 auraloss_module: nn.Module,
```
Comment (zqevans) [inline oobleck/losses.py]: Change to a callable input
